derp/derp_server.go

```diff
@@ -1398,3 +1419,26 @@ func writePublicKey(bw *bufio.Writer, key *key.Public) error {
 	}
 	return nil
 }
+
+type ipBandwidth struct {
+	ip        string
+	bandwidth int64
```

Comment:
bandwidth implies a rate. is this just `bytesTransferred` (or `bytes`)? is this sent or received? or both?

Reply:
will update to bytes sent

Reply:
changed to bytesSent